### DISCUSSION TITLE
the project doesn't know how to run the... [ demo build error ]

### DISCUSSION THREAD

**Discussion by TimerOverflow** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2931] [Q&A] [answered]
hi.

have problem run demo..
i read below manual but, error displayed when hit F5!!

[Compiling From Source](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Compiling-From-Source)

![image](https://user-images.githubusercontent.com/48713974/199868968-2a987dbc-1d10-4b25-9993-97aa64441672.png)

![image](https://user-images.githubusercontent.com/48713974/199869616-5f3e2228-a4e5-4c65-bd91-d308114261a3.png)

installed item

visual studio 2022

![image](https://user-images.githubusercontent.com/48713974/199870569-8cc24ac0-7a31-4c9f-841a-38e92e931844.png)

![image](https://user-images.githubusercontent.com/48713974/199870597-fdf1d182-3c05-4144-beb6-b667aa1a6697.png)

this demo require add Nuget : MaterialDesignThemes ?

**Comment by TimerOverflow** [comment]
build fine.. that's ok.
and work fine debug output.
but, still doesn't run in visual studio debug run,

![image](https://user-images.githubusercontent.com/48713974/199928230-ebb056c8-4099-49a1-bccf-cf243a6b5516.png)

**Comment by Keboo** [comment]
It looks like you may be hitting the same error described here: https://developercommunity.visualstudio.com/t/vs-2022-preview-2-the-project-doesnt-know-how-to-r/1491655.

Can you check within the Visual Studio Installer to see if you have "Development tools for .NET". On my Installer it is labeled as "C# and Visual Basic" under the "Development activities"
![image](https://user-images.githubusercontent.com/952248/200107343-3f9645e8-9083-4824-a225-fee84145ae96.png)

**Reply by TimerOverflow** [reply]
hi. Keboo.
thank's for reply.

![image](https://user-images.githubusercontent.com/48713974/200205133-234339d5-816b-4936-9f84-8f97e8a1e523.png)

my visual studio too istalled, "C# and Visual Basic". and there is no name of item "Development tools for .NET"

and still doesn't work debug run..

**Reply by Keboo** [reply]
Are you able to create a new WPF project and have it run?

**Comment by TimerOverflow** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/48713974/200232671-415dee9b-fb9c-48ac-8ed9-2e3a704af69d.png)

![image](https://user-images.githubusercontent.com/48713974/200232703-7b3fe156-4bf3-460e-84b7-4c423958b7db.png)

oh..
same problem.

Apparently there were no problems when I recently made it with the same template. So I doubted the Material Project. sorry. Apparently there is a problem with my visual studio. thank you
